src/core/gps/qgsgpsconnection.h

```diff
@@ -128,6 +128,12 @@ class CORE_EXPORT QgsGpsInformation
      */
     double elevation = 0;
 
+    /**
+     * Geoidal separation (Diff. between WGS-84 earth ellipsoid and
+     * mean sea level.  -=geoid is below WGS-84 ellipsoid)
+     */
```

Comment:
Since

Reply:
Is "-=" deliberate?

Reply:
It's copy paste. But I struggle to understand this wording ...

https://gis.stackexchange.com/questions/174046/relation-between-geoidal-separation-and-antenna-altitude-in-gga-sentence#:~:text=The%20geoidal%20separation%20just%20reports,height%20above%20mean%20sea%20level.

Reply:
Probably "negative values mean..."

Reply:
Yeah, I think that's right